app/src/main/java/com/fastlink/zekrapp/appData/SingletonModule.kt

```diff
@@ -0,0 +1,24 @@
+package com.fastlink.zekrapp.appData
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object SingletonModule {
```

Comment:
SingletonModule doesn't mean anything. Name it AppModule. Also this has nothing to do with data, so don't place it in the data package.